Use the new /run paths and use /usr/(s)bin for Fedora 17's /usr move.
diff -Nru laptop-mode-tools_1.61.orig/usr/sbin/laptop_mode laptop-mode-tools_1.61/usr/sbin/laptop_mode
--- laptop-mode-tools_1.61.orig/usr/sbin/laptop_mode	2012-09-06 16:33:16.430459219 -0400
+++ laptop-mode-tools_1.61/usr/sbin/laptop_mode	2012-09-06 16:44:50.119259032 -0400
@@ -144,9 +144,9 @@
 # We don't want to discard all the remaining 4 events, but at least honor 1 so that we can act to the changes
 # that occured in the last 7 seconds.
 # So, Event-2 acquire LMT_REQ_LOCK and waits to acquire LMT_INVOC_LOCK (which is acquire by Event-1)
-LMT_REQ_LOCK="/var/lock/lmt-req.lock"
-LMT_INVOC_LOCK="/var/lock/lmt-invoc.lock"
-LMT_BATTPOLL_LOCK="/var/lock/lmt-battpoll.lock"
+LMT_REQ_LOCK="/run/lock/lmt-req.lock"
+LMT_INVOC_LOCK="/run/lock/lmt-invoc.lock"
+LMT_BATTPOLL_LOCK="/run/lock/lmt-battpoll.lock"
 FLOCK=`which flock`
 
 checkint ()
@@ -536,10 +536,10 @@
 		    fi
 	    done
 	    log "STATUS" " "
-	    if [ -e /var/run/laptop-mode-tools/enabled ] ; then
-		    log "STATUS" "Laptop Mode Tools is allowed to run: /var/run/laptop-mode-tools/enabled exists."
+	    if [ -e /run/laptop-mode-tools/enabled ] ; then
+		    log "STATUS" "Laptop Mode Tools is allowed to run: /run/laptop-mode-tools/enabled exists."
 	    else
-		    log "STATUS" "Laptop Mode Tools is NOT allowed to run: /var/run/laptop-mode-tools/enabled does not exist."
+		    log "STATUS" "Laptop Mode Tools is NOT allowed to run: /run/laptop-mode-tools/enabled does not exist."
 	    fi
 	log "STATUS" " "
 	    STATFILES="/proc/sys/vm/laptop_mode /proc/apm /proc/pmu/info /proc/sys/vm/bdflush /proc/sys/vm/dirty_ratio /proc/sys/fs/xfs/age_buffer /proc/sys/fs/xfs/sync_interval /proc/sys/fs/xfs/lm_age_buffer /proc/sys/fs/xfs/lm_sync_interval /proc/sys/vm/pagebuf/lm_flush_age /proc/sys/fs/xfs/xfsbufd_centisecs /proc/sys/fs/xfs/xfssyncd_centisecs /proc/sys/vm/dirty_background_ratio /proc/sys/vm/dirty_expire_centisecs /proc/sys/fs/xfs/age_buffer/centisecs /proc/sys/vm/dirty_writeback_centisecs /sys/devices/system/cpu/*/cpufreq/cpuinfo_*_freq /sys/devices/system/cpu/*/cpufreq/scaling_governor /proc/acpi/button/lid/*/state /proc/acpi/ac_adapter/*/state /proc/acpi/battery/*/state /sys/class/power_supply/*/online /sys/class/power_supply/*/state"
@@ -615,7 +615,7 @@
 	    shift
     done
 
-    mkdir -p /var/run/laptop-mode-tools
+    mkdir -p /run/laptop-mode-tools
 
     # Used to display laptop mode state later on. This is the enabled/disabled 
     # state for laptop mode processing, it tells us nothing about whether laptop
@@ -670,8 +670,8 @@
 
     # If the init script has not been run or has been run with the "stop"
     # argument, then we should never start laptop mode.
-    if [ ! -f /var/run/laptop-mode-tools/enabled ] ; then
-	    log "VERBOSE" "Laptop mode disabled because /var/run/laptop-mode-tools/enabled is missing."
+    if [ ! -f /run/laptop-mode-tools/enabled ] ; then
+	    log "VERBOSE" "Laptop mode disabled because /run/laptop-mode-tools/enabled is missing."
 	    STATE=disabled
     fi
 
@@ -1003,12 +1003,12 @@
 	    log "MSG" "Laptop mode "
     fi
 
-    # WAS_ACTIVE is used later on. If there is no /var/run/laptop-mode-tools/state, then
+    # WAS_ACTIVE is used later on. If there is no /run/laptop-mode-tools/state, then
     # we know that laptop mode wasn't active before.
     WAS_ACTIVE=0
     log "VERBOSE" "Checking if desired state is different from current state."
-    if [ -f /var/run/laptop-mode-tools/state ] ; then
-	    read WAS_ACTIVE WAS_ON_AC WAS_ACTIVATE_WITH_POSSIBLE_DATA_LOSS WAS_STATE < /var/run/laptop-mode-tools/state
+    if [ -f /run/laptop-mode-tools/state ] ; then
+	    read WAS_ACTIVE WAS_ON_AC WAS_ACTIVATE_WITH_POSSIBLE_DATA_LOSS WAS_STATE < /run/laptop-mode-tools/state
 	    if [ "$WAS_STATE" != "" ] ; then
 		    if [ "$WAS_ACTIVE" -eq "$ACTIVATE" -a "$WAS_ON_AC" -eq "$ON_AC" -a "$WAS_ACTIVATE_WITH_POSSIBLE_DATA_LOSS" -eq "$ACTIVATE_WITH_POSSIBLE_DATA_LOSS" -a "$WAS_STATE" = "$STATE" -a "$FORCE" -eq 0 ] ; then
 			    log "MSG" "$STATE, "
@@ -1024,9 +1024,9 @@
 		    fi
 	    fi
     else
-	    log "VERBOSE" "/var/run/laptop-mode-tools/state does not exist, no previous state."
+	    log "VERBOSE" "/run/laptop-mode-tools/state does not exist, no previous state."
     fi
-    echo "$ACTIVATE $ON_AC $ACTIVATE_WITH_POSSIBLE_DATA_LOSS $STATE" > /var/run/laptop-mode-tools/state
+    echo "$ACTIVATE $ON_AC $ACTIVATE_WITH_POSSIBLE_DATA_LOSS $STATE" > /run/laptop-mode-tools/state
 
     if [ "$ACTIVATE" -eq 1 ] ; then
 	    log "MSG" "$STATE, active"
diff -Nru laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/cpufreq laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/cpufreq
--- laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/cpufreq	2012-09-06 16:33:16.429459220 -0400
+++ laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/cpufreq	2012-09-06 16:41:35.929215938 -0400
@@ -80,7 +80,7 @@
 			log "VERBOSE" "Setting CPU minimum frequency for cpu $THISCPU to $THIS_CPU_MINFREQ."
 			set_sysctl $THISCPU/cpufreq/scaling_min_freq $THIS_CPU_MINFREQ
 			log "VERBOSE" "Setting CPU frequency governor for cpu $THISCPU to $THIS_CPU_GOVERNOR."
-			log "VERBOSE" "`/sbin/modprobe -q cpufreq_$THIS_CPU_GOVERNOR 2>&1`"
+			log "VERBOSE" "`/usr/sbin/modprobe -q cpufreq_$THIS_CPU_GOVERNOR 2>&1`"
 			set_sysctl $THISCPU/cpufreq/scaling_governor $THIS_CPU_GOVERNOR
 
 			# Retain for backward compatibility
diff -Nru laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/hdparm laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/hdparm
--- laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/hdparm	2012-09-06 16:33:16.429459220 -0400
+++ laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/hdparm	2012-09-06 16:40:07.691258269 -0400
@@ -34,7 +34,7 @@
 	# It assumes more or less recent udev (> 070)
 	if [ $HAVE_UDEVINFO -ne 0 ] ; then
 		log "VERBOSE" "Querying $1 media type using udevinfo: "
-		if [ -x /sbin/udevadm ]; then
+		if [ -x /usr/sbin/udevadm ]; then
 			eval "$(udevadm info -q env -n $1 | egrep '(ID_TYPE=|ID_BUS=)' )"
 		else
 			eval "$(udevinfo -q env -n $1 | egrep '(ID_TYPE=|ID_BUS=)' )"
diff -Nru laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/laptop-mode laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/laptop-mode
--- laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/laptop-mode	2012-09-06 16:33:16.429459220 -0400
+++ laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/laptop-mode	2012-09-06 16:46:47.071290549 -0400
@@ -115,7 +115,7 @@
 
 
 if [ "$CONTROL_READAHEAD" -ne 0 ] ; then
-	if /sbin/blockdev --help 2>&1 | grep -Fq -- '--setfra' ; then
+	if /usr/sbin/blockdev --help 2>&1 | grep -Fq -- '--setfra' ; then
 		READAHEAD_OPTION=--setfra					
 	else
 		READAHEAD_OPTION=--setra
@@ -271,13 +271,13 @@
 				log "VERBOSE" "Original options: $OPTS"
 				if [ "$WAS_ACTIVE" -eq 0 ] ; then
 					# Coming from inactive state: save last known mount options for the device.
-					log "VERBOSE" "Updating /var/run/laptop-mode-tools/nolm-mountopts." 
-					if [ -f /var/run/laptop-mode-tools/nolm-mountopts ] ; then 
-						sed -i "s|^$DEV .*$||" /var/run/laptop-mode-tools/nolm-mountopts
+					log "VERBOSE" "Updating /run/laptop-mode-tools/nolm-mountopts." 
+					if [ -f /run/laptop-mode-tools/nolm-mountopts ] ; then 
+						sed -i "s|^$DEV .*$||" /run/laptop-mode-tools/nolm-mountopts
 					fi
-					echo $DEV $OPTS >> /var/run/laptop-mode-tools/nolm-mountopts
+					echo $DEV $OPTS >> /run/laptop-mode-tools/nolm-mountopts
 				else
-					log "VERBOSE" "Not updating /var/run/laptop-mode-tools/nolm-mountopts because laptop mode was already active."
+					log "VERBOSE" "Not updating /run/laptop-mode-tools/nolm-mountopts because laptop mode was already active."
 				fi
 				if [ "$FST" = 'unknown' ]; then
 					log "VERBOSE" "Deducing fstype for $MP."
@@ -312,8 +312,8 @@
 						;;
 				esac
 				if [ -b $DEV -a "$CONTROL_READAHEAD" -ne 0 ] ; then
-					log "VERBOSE" "Executing: /sbin/blockdev $READAHEAD_OPTION $(($LM_READAHEAD * 2)) $DEV"
-					log "VERBOSE" "`/sbin/blockdev $READAHEAD_OPTION $(($LM_READAHEAD * 2)) $DEV 2>&1`"
+					log "VERBOSE" "Executing: /usr/sbin/blockdev $READAHEAD_OPTION $(($LM_READAHEAD * 2)) $DEV"
+					log "VERBOSE" "`/usr/sbin/blockdev $READAHEAD_OPTION $(($LM_READAHEAD * 2)) $DEV 2>&1`"
 				fi
 			fi
 		done
@@ -422,8 +422,8 @@
 				# Retrieve original non-laptop mode mount options and restore them.
 				# If the file that stores them doesn't exist, then laptop mode
 				# has never been started.
-				if [ "$WAS_ACTIVE" -ne 0 -a -f /var/run/laptop-mode-tools/nolm-mountopts ] ; then						
-					SAVED_OPTS=`grep "^$DEV " /var/run/laptop-mode-tools/nolm-mountopts`
+				if [ "$WAS_ACTIVE" -ne 0 -a -f /run/laptop-mode-tools/nolm-mountopts ] ; then						
+					SAVED_OPTS=`grep "^$DEV " /run/laptop-mode-tools/nolm-mountopts`
 					SAVED_OPTS=${SAVED_OPTS#* } # trim device name
 				
 					case "$FST" in
@@ -444,8 +444,8 @@
 					log "VERBOSE" "Not remounting."
 				fi
 				if [ -b $DEV -a "$CONTROL_READAHEAD" -ne 0 ] ; then
-					log "VERBOSE" "Executing: /sbin/blockdev $READAHEAD_OPTION $(($NOLM_READAHEAD * 2)) $DEV"
-					log "VERBOSE" "`/sbin/blockdev $READAHEAD_OPTION $(($NOLM_READAHEAD * 2)) $DEV 2>&1`"
+					log "VERBOSE" "Executing: /usr/sbin/blockdev $READAHEAD_OPTION $(($NOLM_READAHEAD * 2)) $DEV"
+					log "VERBOSE" "`/usr/sbin/blockdev $READAHEAD_OPTION $(($NOLM_READAHEAD * 2)) $DEV 2>&1`"
 				fi
 			fi
 		done
diff -Nru laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/lcd-brightness laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/lcd-brightness
--- laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/lcd-brightness	2012-09-06 16:33:16.429459220 -0400
+++ laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/lcd-brightness	2012-09-06 16:45:30.116271862 -0400
@@ -14,8 +14,8 @@
 	else
 		BRIGHTNESS_COMMAND="$BATT_BRIGHTNESS_COMMAND"
 	fi
-	if [ -f /var/run/laptop-mode-tools/state-brightness-command ] ; then
-		OLD_BRIGHTNESS_COMMAND="$( cat /var/run/laptop-mode-tools/state-brightness-command )"
+	if [ -f /run/laptop-mode-tools/state-brightness-command ] ; then
+		OLD_BRIGHTNESS_COMMAND="$( cat /run/laptop-mode-tools/state-brightness-command )"
 	fi
 	
 	if [ "$FORCE" -ne 0 -o "$BRIGHTNESS_COMMAND" != "$OLD_BRIGHTNESS_COMMAND" ] ; then
@@ -27,7 +27,7 @@
 			log "ERR" "Please check your configuration in /etc/laptop-mode/conf.d/lcd-brightness.conf."
 			log "ERR" "Command failed."
 		fi
-		echo -n $BRIGHTNESS_COMMAND > /var/run/laptop-mode-tools/state-brightness-command
+		echo -n $BRIGHTNESS_COMMAND > /run/laptop-mode-tools/state-brightness-command
 	else
 		log "VERBOSE" "Not setting LCD brightness -- command was the same as last time."
 	fi
diff -Nru laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/start-stop-programs laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/start-stop-programs
--- laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/start-stop-programs	2012-09-06 16:33:16.429459220 -0400
+++ laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/start-stop-programs	2012-09-06 16:49:02.349307297 -0400
@@ -7,8 +7,8 @@
    #
    # Undo the previous state.
    #
-   if [ -f /var/run/laptop-mode-tools/start-stop-undo-actions ] ; then
-	cat /var/run/laptop-mode-tools/start-stop-undo-actions | \
+   if [ -f /run/laptop-mode-tools/start-stop-undo-actions ] ; then
+	cat /run/laptop-mode-tools/start-stop-undo-actions | \
 		while read SERVICE STARTSTOPACTION ; do
 			systemctl $STARTSTOPACTION $SERVICE
 		done
@@ -23,7 +23,7 @@
 	# next state change. Note: we actually
 	# write the actions to the file in reverse order,
 	# so we can execute the commands easily afterwards.
-	echo > /var/run/laptop-mode-tools/start-stop-undo-actions 
+	echo > /run/laptop-mode-tools/start-stop-undo-actions 
 		
 	   
 	if [ $ON_AC -eq 1 ] ; then
@@ -50,7 +50,7 @@
 				SERVICE=$(readlink -f "$SERVICE")
 				SERVICE=$(basename "$SERVICE")
 				systemctl stop "$SERVICE"
-				sed -i "1i $SERVICE start" /var/run/laptop-mode-tools/start-stop-undo-actions
+				sed -i "1i $SERVICE start" /run/laptop-mode-tools/start-stop-undo-actions
 			fi
 		done
 	fi
@@ -61,7 +61,7 @@
 				SERVICE=$(readlink -f "$SERVICE")
 				SERVICE=$(basename "$SERVICE")
 				systemctl start "$SERVICE"
-				sed -i "1i $SERVICE stop" /var/run/laptop-mode-tools/start-stop-undo-actions
+				sed -i "1i $SERVICE stop" /run/laptop-mode-tools/start-stop-undo-actions
 			fi
 		done
 	fi
@@ -75,12 +75,12 @@
 		for SERVICE in $STOP_SERVICES ; do
 			log "VERBOSE" "Stopping service $SERVICE."
 			systemctl stop $SERVICE
-			sed -i "1i $SERVICE start" /var/run/laptop-mode-tools/start-stop-undo-actions
+			sed -i "1i $SERVICE start" /run/laptop-mode-tools/start-stop-undo-actions
 		done
 		for SERVICE in $START_SERVICES ; do
 			log "VERBOSE" "Starting service $SERVICE."
 			systemctl start $SERVICE
-			sed -i "1i $SERVICE stop" /var/run/laptop-mode-tools/start-stop-undo-actions
+			sed -i "1i $SERVICE stop" /run/laptop-mode-tools/start-stop-undo-actions
 		done
 	fi
    fi
diff -Nru laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/video-out laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/video-out
--- laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/video-out	2012-09-06 16:33:16.429459220 -0400
+++ laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/video-out	2012-09-06 16:46:03.786280944 -0400
@@ -27,17 +27,17 @@
 		DISABLE_VIDEO_OUTPUTS="$BATT_DISABLE_VIDEO_OUTPUTS"
 	fi
 
-	if [ -f /var/run/laptop-mode-tools/disabled-video-outputs ] ; then
-		cat /var/run/laptop-mode-tools/disabled-video-outputs |
+	if [ -f /run/laptop-mode-tools/disabled-video-outputs ] ; then
+		cat /run/laptop-mode-tools/disabled-video-outputs |
 			while read VIDEO_OUTPUT REMAINDER ; do
 				do_xrandr $VIDEO_OUTPUT auto
 			done
 	fi
 			
-	rm -f /var/run/laptop-mode-tools/disabled-video-outputs
+	rm -f /run/laptop-mode-tools/disabled-video-outputs
 	
 	for VIDEO_OUTPUT in $DISABLE_VIDEO_OUTPUTS ; do
-		echo $VIDEO_OUTPUT >> /var/run/laptop-mode-tools/disabled-video-outputs
+		echo $VIDEO_OUTPUT >> /run/laptop-mode-tools/disabled-video-outputs
 		
 		do_xrandr $VIDEO_OUTPUT off
 	done
diff -Nru laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/wireless-ipw-power laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/wireless-ipw-power
--- laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/wireless-ipw-power	2012-09-06 16:33:16.429459220 -0400
+++ laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/wireless-ipw-power	2012-09-06 16:38:24.776307951 -0400
@@ -21,16 +21,12 @@
 	I2100_DRIVERNAME=ipw2100
 
 	# find executables
-	if [ -x /sbin/iwpriv ] ; then
-		IWPRIV=/sbin/iwpriv
-	elif [ -x /usr/sbin/iwpriv ] ; then
+	if [ -x /usr/sbin/iwpriv ] ; then
 		IWPRIV=/usr/sbin/iwpriv
 	else
 		log "VERBOSE" "iwpriv is not installed"
 	fi
-	if [ -x /sbin/iwconfig ] ; then
-		IWCONFIG=/sbin/iwconfig
-	elif [ -x /usr/sbin/iwconfig ] ; then
+	if [ -x /usr/sbin/iwconfig ] ; then
 		IWCONFIG=/usr/sbin/iwconfig
 	else
 		log "VERBOSE" "iwconfig is not installed"
diff -Nru laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/wireless-iwl-power laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/wireless-iwl-power
--- laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/wireless-iwl-power	2012-09-06 16:33:16.429459220 -0400
+++ laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/wireless-iwl-power	2012-09-06 16:39:44.808269283 -0400
@@ -57,16 +57,12 @@
 	[ "$IWL_BATT_POWER" ] || IWL_BATT_POWER=3
 
 	# find executables
-	if [ -x /sbin/iwpriv ] ; then
-		IWPRIV=/sbin/iwpriv
-	elif [ -x /usr/sbin/iwpriv ] ; then
+	if [ -x /usr/sbin/iwpriv ] ; then
 		IWPRIV=/usr/sbin/iwpriv
 	else
 		log "VERBOSE" "iwpriv is not installed"
 	fi
-	if [ -x /sbin/iwconfig ] ; then
-		IWCONFIG=/sbin/iwconfig
-	elif [ -x /usr/sbin/iwconfig ] ; then
+	if [ -x /usr/sbin/iwconfig ] ; then
 		IWCONFIG=/usr/sbin/iwconfig
 	else
 		log "VERBOSE" "iwconfig is not installed"
diff -Nru laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/wireless-power laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/wireless-power
--- laptop-mode-tools_1.61.orig/usr/share/laptop-mode-tools/modules/wireless-power	2012-09-06 16:33:16.429459220 -0400
+++ laptop-mode-tools_1.61/usr/share/laptop-mode-tools/modules/wireless-power	2012-09-06 16:40:33.519245853 -0400
@@ -56,9 +56,7 @@
 	[ "$WIRELESS_BATT_POWER_SAVING" ] || WIRELESS_BATT_POWER_SAVING=1
 
 	# Find executables
-	if [ -x /sbin/iwconfig ] ; then
-		IWCONFIG=/sbin/iwconfig
-	elif [ -x /usr/sbin/iwconfig ] ; then
+	if [ -x /usr/sbin/iwconfig ] ; then
 		IWCONFIG=/usr/sbin/iwconfig
 	else
 		log "VERBOSE" "iwconfig is not installed"
